structure_tests/ext_run.sh

```diff
@@ -42,11 +44,13 @@ usage() {
 helper() {
 	echo "$USAGE_STRING"
 	echo
-	echo "	-i, --image          image to run tests on"
-	echo "	-c, --config         path to json/yaml config file"
-	echo "	-v                   display verbose testing output"
-	echo "	-e, --entrypoint     specify custom docker entrypoint for image"
-	echo "	--no-pull            don't pull latest structure test image"
+	echo "    -i, --image          image to run tests on"
+	echo "    -c, --config         path to json/yaml config file"
+	echo "    -w, --workspace      path to directory to be mounted as"
```

Comment:
This makes sense, could you give an example of usage, though? It could be a bit confusing.